### DISCUSSION TITLE
Jetty 10 / Eclipse 2022-09

### DISCUSSION THREAD

**Discussion by N1k145** [eclipse-glsp/glsp#763] [Q&A] [answered]
Hey,
I'm currently trying to integrate GLSP (server + ide) into an Eclipse 2022-09 based product.
From what I have seen, eclipse switched to Jetty 10, including the related dependencies.

Is there a plan to updated GLSP to Jetty 10 to make it compatible with the current Eclipse IDE version?

**Comment by martin-fleck-at** [comment, marked as answer]
@N1k145 Thank you for raising this! I was not aware that we lost compatibility with the latest version of Eclipse so I opened a new issue: https://github.com/eclipse-glsp/glsp/issues/764. We need to see how to best deal with different Eclipse versions and the update might not be very straight forwards as the [glsp-modelserver-integration](https://github.com/eclipse-emfcloud/modelserver-glsp-integration) also depends on it and the modelserver is also using a specific version of Jetty. Nevertheless, I'll raise it with the team and hopefully we can do that soon.

**Reply by N1k145** [reply]
@martin-fleck-at thanks for looking into it.
I can't garantee that it is not possible to have both versions of jetty in Eclipse, but it doesn't feel right
